notifications-backend/pom.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>notifications-backend</artifactId>
+
+    <parent>
+        <groupId>com.redhat.cloud.notifications</groupId>
+        <artifactId>notifications-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <properties>
+        <mockserver-client-java.version>5.5.4</mockserver-client-java.version> <!-- not the newest, but matches what is in org.testcontainers:mockserver -->
+        <openapi-parser.version>4.0.4</openapi-parser.version>
+    </properties>
+
```

Comment:
Do the plugins declared in the parent pom really work here? Declaring everything at parent level and nothing at child level seems a bit unusual, but I'm not a Maven expert so maybe that works.

Reply:
I took a look at policies-engine and it's pretty much the same there. At least I didn't find plugins that are in the parent as well as in the child pom. My expectation it, that the plugins and the dependencies gets inherited to the child projects

Reply:
WTF...

Reply:
I just moved all the plugins in the child pom. I would like to finish this PR and test it later when the second projects gets added to it.

Reply:
By putting it in the child pom it recognizes checkstyle issues.